pkgs/applications/misc/sfm/default.nix

```
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     hash = "sha256-DwXKrSqcebNI5N9REXyMV16W2kr72IH9+sKSVehc5zw=";
   };
 
-  configFile = lib.optionalString (conf!=null) (lib.writeText "config.def.h" conf);
+  configFile = lib.optionalString (conf!=null) (builtins.toFile "config.def.h" conf);
```

```suggestion
  configFile = lib.optionalString (conf!=null) (writeText "config.def.h" conf);
```
and add it to inputs.